gosling/schema/channels.py

```diff
@@ -424,6 +424,34 @@ def __init__(self, value, **kwds):
         super(TextValue, self).__init__(value=value, **kwds)
 
 
+class Tooltip(FieldChannelMixin, core.Tooltip):
+    """Tooltip schema wrapper
+
+    Mapping(required=[shorthand])
+
+    Attributes
+    ----------
+
+    shorthand : string
+        shorthand for field, aggregate, and type
+    alt : string
+
```

Comment:
Just curious since comments are missing and my understanding is that this file is auto-generated – Do I need to add comments to the schema of gosling.js?

Reply:
Yes, JSDoc comments `/** ... */` added before each field here,

https://github.com/gosling-lang/gosling.js/blob/24148ae3ffc5b7c0e0d4462b490d08bb2ab44fe0/src/core/gosling.schema.ts#L333-L338

will but added to the generated `channels.py` if provided.

Reply:
Additionally, I found out that (unforutnately) since we use an intersection type to define `SingleTrack`:

```typescript
export type SingleTrack = SingleTrackBase & Encoding;
```

the `Encoding` definition doesn't end up in the generated schema, so the `tools/generate_schema_wrapper.py` has to inspect the fields on `SingleTrack` to "guess" whether they are an encoding type or not. This isn't a huge worry, but could be improved if we ended up creating a `encoding` field for `SingleTrack` rather than a flat object with some encodings and some properties.

Reply:
Okay. Since having an `encoding` will make breaking changes and the issue does not seem to be critical, we can improve this later when we want to make a major version bump.